docs/swarm/join-nodes.md

```diff
@@ -29,6 +29,11 @@ also run tasks.
 Before you add nodes to a swarm you must install Docker Engine 1.12 or later on
 the host machine.
 
+The Docker Engine joins the swarm according to the **join-token** you provide to
```

Comment:
Maybe add: `Providing the manager token will ensure the node will participate in the swarm as a Manager, and will increase the number of nodes participating in the raft quorum. See here for details on why multiple managers are needed to provide High Availability guarantees to you swarm`